yoga/YGMacros.h

```diff
@@ -38,3 +38,10 @@
 #define YG_ENUM_BEGIN(name) enum name
 #define YG_ENUM_END(name) name
 #endif
+
+#if !defined(TRUE)
+#define TRUE 1==1
```

Comment:
What you need here is simply a non-zero integer value.
```suggestion
#define TRUE 1
```